config.json

```diff
@@ -2338,6 +2338,21 @@
         ],
         "difficulty": 5
       },
+      {
+        "slug": "baffling-birthdays",
+        "name": "Baffling Birthdays",
+        "uuid": "e5ebca45-1e54-45b6-89bb-0cddfd23b6e1",
+        "practices": [
+          "randomness"
+        ],
+        "prerequisites": [
+          "randomness",
+          "dates-and-time",
+          "lists",
+          "enum"
```

Comment:
You have also used: list-comprehensions, ranges, tuples. I think list-comprehensions might be optional (you're not using cartesian products, right?), but ranges might be necessary for generating a list of a given length.

Additionally a MapSet, but we don't have a concept for that 🤷

Reply:
Thank you, I added them all, it could help to know the concepts.

> Additionally a MapSet, but we don't have a concept for that 🤷

I made a [set concept for Elm](https://exercism.org/tracks/elm/concepts/set) a while ago. Want me to fork it? :)

Reply:
Sure thing, if you have the time, it would be very nice to have that concept in Elixir too